auth/ante.go

```diff
@@ -26,7 +26,8 @@ var (
 	gasWantedPerCheckpoinTx sdk.Gas = 10000000
 	gasUsedPerCheckpointTx  sdk.Gas = gasWantedPerCheckpoinTx - 1000000
 
-	feeWantedPerTx = types.Coins{types.Coin{Denom: "vetic", Amount: types.NewInt(1)}}
+	// FeeWantedPerTx fee wanted per tx
+	FeeWantedPerTx = types.Coins{types.Coin{Denom: "vetic", Amount: types.NewInt(1)}}
```

Comment:
rename to `gas` to remove confusion

Reply:
That's not gas. That's fee per tx only. Gas needs price to get actual fee per tx.